### PR TITLE
Feature/comparison details logic

### PR DESCRIPTION
**Pull request by Codo-Baggins**
## What's this PR do?
- Adds logical operations to the ComparisonDetails page to dynamically create blurbs of text that compare deaths of COVID-19 to some user-selected topic. 
- Changes the way the ComparisonDetails info is being displayed.  A ComparisonCard component will be used to display various factual comparisons.  

## Where should the reviewer start?
- The reviewer should start in ComparisonDetails lines 21-36 with the function declarations.  Note that these functions will return data values that will be inserted into interpolated strings to then be displayed on the page.  

## How should this be manually tested?
- Clone down the repo
- `cd` into the cloned down directory
- Run `npm start` in the command line and navigate to the `localhost` URL that it provides you 
- Scroll down the page and click on the dropdown menus
- Select a category and then click the 'View Comparison' button
- You will be brought to a new page with the selected comparison category data
- Observe the data on the page and how no matter which category is selected, the text makes sense

## Any background context you want to provide?
- The ComparisonCards that we have created are not yet receiving any props so we are not utilizing them yet.

### CODE REVIEW

File: src/ComparisonDetails/ComparisonDetails.scss [hunk 1]
```diff
@@ -3,8 +3,14 @@
   flex-direction: column;
   font-family: 'Heebo', sans-serif;
   margin: 10px;
```
Comment (alyssajoyner) [inline src/ComparisonDetails/ComparisonDetails.scss:5]: We should change this px to em to be consistent in our styling.

File: src/ComparisonDetails/ComparisonDetails.scss [hunk 2]
```diff
@@ -3,8 +3,14 @@
   flex-direction: column;
```
Comment (kncaputo) [inline src/ComparisonDetails/ComparisonDetails.scss:3]: We should remember to add sass in here

File: src/ComparisonDetails/ComparisonDetails.tsx [hunk 3]
```diff
@@ -1,22 +1,76 @@
...
+      let deathRatio = (prop.usaStateDeaths / prop.selection.data.deaths)
+      return generateComparisonText(deathRatio)
+    }
+  }
+
+  const generateComparisonText = (deathRatio: number) => {
+    if(deathRatio < 1) {
+      return(
+        <section className='comparison-text'>
+          <p>Despite the staggering number of COVID-19 deaths in 2020, COVID-19
```
Comment (alyssajoyner) [inline src/ComparisonDetails/ComparisonDetails.tsx:40]: We will need to revisit this wording to incorporate the additional information that was put into the data file (i.e. tagline)

File: src/comparisonData.tsx [hunk 4]
```diff
@@ -13,52 +13,65 @@ export interface Comparison {
...
   {
-    id: 2,
+    id: 1,
     category: 'sept11', 
     data: {
-      title: '9/11 Deaths',
-      deaths: 2977,
-      image: ''
+      title: 'September 11th Deaths',
+      subtitle: 'September 11th',
```
Comment (alyssajoyner) [inline src/comparisonData.tsx:20]: Love the additional of the subtitle - this will make formatting easier.

**Review by kncaputo — APPROVED** [review]
Looks good!

**Review by alyssajoyner — APPROVED** [review]
Nice job on this branch - this is the heart of our functionality!

**Pull request merged**